docs/source/quantization.rst

```diff
@@ -186,7 +188,7 @@ Layers for the quantization-aware training
 Observers for computing the quantization parameters
 
 * :class:`~torch.quantization.MinMaxObserver` — Derives the quantization parameters from the running minimum and maximum of the observed tensor inputs (per tensor variant)
-* :class:`~torch.quantization.MovingAverageObserver` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
+* ``MovingAverageObserver`` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
```

Comment:
This doesn't exist in the documentation. I've filed an issue at https://github.com/pytorch/pytorch/issues/27849. There is a comment about it in a section below; I couldn't figure out how to insert a comment here without breaking up the bullet pointing.